src/integrationTest/resources/zipcontents/missing_ocr_data/metadatajson.json

```diff
@@ -0,0 +1,21 @@
+{
+  "po_box": "SSCSPO",
+  "jurisdiction": "SSCS",
+  "delivery_date": "2018-06-23T00:00:00.000Z",
+  "opening_date": "2018-06-24T00:00:00.000Z",
+  "zip_file_createddate": "2018-02-24T00:00:00.000Z",
+  "zip_file_name": "8_24-06-2018-00-00-00.zip",
+  "envelope_classification": "new_application",
+  "scannable_items": [
```

Comment:
Maybe it would also be good to test a case when there's no SSCS1 document among scannable items (which can contain OCR data)?

Reply:
👍